### DISCUSSION TITLE
Layui 表格组件有没有条件格式功能？

### DISCUSSION THREAD

**Discussion by HuangLei0x2e5** [layui/layui#1673] [问答]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

功能请求

### 使用版本

v2.9.6

### 问题描述

大家好，我是新手，想请问一下，Layui的表格组件有没有条件格式功能，比如我用Table来显示设备列表，正常设备显示绿色，告警设备显示黄色，故障设备显示红色

我知道可以自己写JS代码来实现，但是我对前端开发不熟悉，而且感觉这个功能会比较实用，如果有的话，会很方便的。
请高手指教，谢谢。

### 业务代码

```auto
无
```

### 浏览器

chrome 122.0.6261.70

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by Sight-wcg** [comment]
可以使用 table 列属性中的 templet 实现格式化，具体用法参考文档示例。

**Comment by HuangLei0x2e5** [comment]
收到，非常感谢

发件人: morning-star ***@***.***>
发送时间: 2024年3月1日 16:34
收件人: layui/layui ***@***.***>
抄送: HuangLei0x2e5 ***@***.***>; Author ***@***.***>
主题: Re: [layui/layui] Layui 表格组件有没有条件格式功能？ (Issue #1672)


可以使用 table 列属性中的 templet 实现格式化，具体用法参考文档示例。

―
